src/pages/agent/baseline-build-modal/baseline-build-modal.tsx

```diff
@@ -98,7 +99,7 @@ export const BaselineBuildModal = () => {
                 }
                 closeModal();
               }}
-              disabled={(!isConfirmed && !isBaseline) || isLoading}
+              disabled={(!isConfirmed && !isBaseline) || isLoading || status === AGENT_STATUS.OFFLINE}
```

Comment:
```suggestion
              disabled={(!isConfirmed && !isBaseline) || isLoading || status !== AGENT_STATUS.ONLINE}
```
Agent also can be busy

Reply:
Agent can be busy when we start a new build